<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowseFilesDialog</class>
 <widget class="QDialog" name="BrowseFilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="treeView_dirs">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_files">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="action_download_file">
   <property name="text">
    <string>下载</string>
   </property>
  </action>
  <action name="action_upload_file">
   <property name="text">
    <string>上传</string>
   </property>
   <property name="toolTip">
    <string>上传</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
